### PR TITLE
Tidy 1

### PR DESCRIPTION
**Pull request by junghao**
The build and runner image parameters specified in build.sh don't fit with new Dockerfile's base image. Removing.

### CODE REVIEW

File: cmd/fdsn-quake-consumer/main.go [hunk 1]
```diff
@@ -129,6 +129,7 @@ func (n *notification) Process(msg []byte) error {
 		if err := e.save(); err != nil {
 			return fmt.Errorf("error saving SC3ML %s %s: %w", v.S3.Bucket.Name, v.S3.Object.Key, err)
 		}
+		log.Println("saved", e.PublicID)
```
Comment (wilsonjord) [inline cmd/fdsn-quake-consumer/main.go:132]: suggestion: a space would make the logs easier to read, i.e. `saved<space>`?

**Review by wilsonjord — COMMENTED** [review]
Nice and easy, just one comment from me

File: cmd/fdsn-quake-consumer/main.go (hunk 1 above)
Comment (junghao) [reply cmd/fdsn-quake-consumer/main.go:132]: `Println` separates different parameters with space already.
See https://github.com/golang/go/blob/master/src/fmt/print.go#L1216

Comment (wilsonjord) [reply cmd/fdsn-quake-consumer/main.go:132]: I see, thanks!

**Review by wilsonjord — APPROVED** [review]

**Review by wilsonjord — APPROVED** [review]

**Review by wilsonjord — CHANGES_REQUESTED** [review]

**Review by wilsonjord — APPROVED** [review]

**Pull request merged**